aeron-driver/src/main/java/io/aeron/driver/DriverConductor.java

```diff
@@ -1640,6 +1640,7 @@ private NetworkPublication newNetworkPublication(
                 ctx.retransmitUnicastDelayGenerator(),
                 ctx.retransmitUnicastLingerGenerator(),
                 udpChannel.hasGroupSemantics(),
+                params.hasRetransmitsActiveMax ? params.retransmitsActiveMax : Configuration.retransmitsActiveMax(),
```

Comment:
The fallback value should be derived from the driver context, the `Configuration.retransmitsActiveMax()` should be used to set initial value on the context.

Reply:
Good catch!